sogs/model/room.py

```diff
@@ -17,21 +17,15 @@
     PostRateLimited,
     InvalidData,
 )
+from model.bot import Bot
 
 import os
-import random
 import re
 import sqlalchemy.exc
 import time
 from typing import Optional, Union, List
 
 
-# TODO: These really should be room properties, not random global constants (these
-# are carried over from old SOGS).
-rate_limit_size = 5
-rate_limit_interval = 16.0
```

Comment:
The TODO was poorly worded; it didn't mean properties as in object properties, but rather configurable attributes stored in the room table that an admin could set/update/etc.